tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -142,6 +142,31 @@ public function testRemove()
         $this->assertEquals('defaultValue', $default);
     }
 
+    public function testRemoveValue()
```

Comment:
@dynasource I've added your code for the tests. Thank you very much for the help and explanation.